quantum/keymap_extras/sendstring_uk.h

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2019 Rys Sommefeldt
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* SEND_STRING() LUTs for UK layouts */
+
+#ifndef SENDSTRING_UK
```

Comment:
We can actually replace lines 19 and 20 with a `#pragma once`

Reply:
Actually, for this, this is one of those cases that I think it's not only fine, but actually probably better to have this sort of include guard.